### DISCUSSION TITLE
CNCF projects lists of gsoc - 2023

### DISCUSSION THREAD

**Discussion by Aman123lug** [cncf/mentoring#846] [Q&A] [answered]
CNCF project list for gsoc - 2023 ?

**Comment by aliok** [comment, marked as answer]
@Aman123lug they're here: https://github.com/cncf/mentoring/blob/main/summerofcode/2023.md

**Comment by Aman123lug** [comment]
@aliok i want to work on Armada project for gsoc

**Reply by aliok** [reply]
@Aman123lug 

My recommendations:
- Have a look at Armada documentation, try tutorials, run things on your machine
- Have a look at Armada's "good-first-issue"s
- Reach out to the Armada community
- Check if you have the recommended skills for the project ideas listed for Armada

Good luck!

**Reply by kannon92** [reply]
@Aman123lug thank you for your interest! Please join our slack and @aliok suggestions are good ones!

**Comment by Aman123lug** [comment]
@kannon92 slack link?

**Reply by Amit-unchartered** [reply]
@Aman123lug If you already have an account on CNCF Slack, join #armada on https://cloud-native.slack.com/
                         If you need an inviation to CNCF Slack, you can get one at https://slack.cncf.io/

**Comment by Amit-unchartered** [comment]
@barnettZQG  I want to contribute to the KubeVela Project - Expand multiple database drivers for the API server, for GSOC. Really excited!! Please suggest me how to get there.
